teps/0089-nonfalsifiable-provenance-support.md

```diff
@@ -59,26 +63,41 @@ There are a couple issues with this:
 2. Tekton Pipelines can't verify that the results it reads weren't modified
 
 This is where SPIRE comes in!
-SPIRE can be used to request signatures and certificates (SVIDs) for a given workload.
-This workload can be anything, including a Result, a Pod spec, or an entire TaskRun yaml.
+SPIRE can be used to request JWTs and certificates (SVIDs) for a given workload (pod for kubernetes).
 
 We're going to use SPIRE to mitigate both of the issues mentioned above.
 
 ## SPIRE Concepts
 Before discussing how SPIRE is going to fix these issues, here's a very basic overview of how it works:
 
-Pods running in the cluster can interact with SPIRE.
-A pod will send some `payload` to SPIRE for signing.
-The SPIRE socket will confirm that the request came from an approved workload (in our case, the Tekton controller or Pods created by the controller) and then it will sign the `payload`.
-SPIRE will return:
-1. The `signature` over the `payload`
-1. A SPIFFE Verifiable Identity Document, or [SVID](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-verifiable-identity-document-svid), an X509 certificate containing the public key used to verify the `signature` against the `payload`
```

Comment:
@lumjjb just double checking, you removed this description of SPIRE signing a payload via the SPIRE socket which I assume means SPIRE does not actually do any signing? i.e. it just make the identity doc (SVID) and the private key available, and the caller is then responsible for what happens after that, e.g. doing the actual signing?